src/_includes/partials/footer.njk

```diff
@@ -1,24 +1,20 @@
 <footer>
-  <nav class="flex flex-wrap px-16" role="navigation" aria-label="main navigation">
-    <ul class="w-full sm:w-full md:w-1/2 lg:w-1/2 xl:w-1/2">
-      <li><a href="/">Home</a></li>
-      <li><a href="/product-and-services">Product and Services</a></li>
-      <li><a href="/contributing">Become a Contributor</a></li>
-      <li><a href="/maintaining">Become a Maintainer</a></li>
-      <li><a href="/contact-us">Contact Us</a></li>
-    </ul>
-
-    <ul  class="w-full sm:w-full md:w-1/2 lg:w-1/2 xl:w-1/2">
-      <li><a href="/operating-agreement">Operating Agreement</a></li>
-      <li><a href="/code-of-conduct">Code of Conduct</a></li>
-      <li><a href="/financial-overview">Financial Overview</a></li>
-      <li><a href="/meeting-notes">Meeting Notes</a></li>
-      <li><a href="/project-guidelines">Project Guidelines</a></li>
-    </ul>
-  </nav>
```

Comment:
I strongly believe we want to leave some of this in the footer. Footers are a great place to put "signposts" to useful-but-not-core trails of information.

When we put that kind of thing behind drop-downs; we make it harder for folks who use screen readers to navigate the website.

Could you reach out to @colombene to get her thoughts on how to organize website navigation?

Reply:
Yes, I was under the impression from the last design session that we would like to consolidate them into header and group them under Join us or About us according to the site map but I didn't confirm. I can reach out and get thoughts on this.